build/yaml/sharedResources/createSharedResources.yml

```diff
@@ -7,142 +7,145 @@ trigger: none
 pr: none
 
 pool:
-  vmImage: 'windows-2019'
+  vmImage: "windows-2019"
 
 variables:
-  InternalAppInsightsName: 'bffnappinsights$($env:ResourceSuffix)'
-  InternalAppServicePlanLinuxName: 'bffnbotsappservicelinux$($env:ResourceSuffix)'
-  InternalAppServicePlanWindowsName: 'bffnbotsappservicewin$($env:ResourceSuffix)'
-  InternalCosmosDBName: 'bffnbotstatedb$($env:ResourceSuffix)'
-  InternalKeyVaultName: 'bffnbotkeyvault$($env:ResourceSuffix)'
+  ## Azure Resources (Define these variables in Azure)
+  # AzureSubscription: Service Connection Name to Manage Azure resources.
+  # KeyVaultObjectId: Suscription's Object Id to create the keyvault to store App Registrations.
+  # AppServicePlanPricingTier: (optional) Pricing Tier for App Service Plans, default F1.
+  # ResourceGroupName: (optional) Name of the Resource Group for the shared resources.
+  # ResourceSuffix: (optional) Suffix to add to the resources' name to avoid collitions.
+
+  ## Internal variables
+  InternalAppInsightsName: "bffnappinsights$($env:ResourceSuffix)"
```

Comment:
We need the $env: for the optional variables that may not be configured in the pipeline. Otherwise, it resolves the value as the literal $(RESOURCESUFFIX). We changed all variables to uppercase so it works on Linux too.